build/scripts/configs/makeBaseConfig.ts

```diff
@@ -124,6 +124,7 @@ ${chalk.green(aliases)}`;
         },
         performance: { hints: false },
         plugins: [
+            new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en|es|fr|pt|ch/),
```

Comment:
what we are trying to get here? Where this list of locales came from?
I don't see any list in original issue.